tests.py

```diff
@@ -73,9 +74,8 @@ def setUpClass(cls):
     cls.docker_directory_path = os.path.join('test_data', 'docker')
     if not os.path.isdir(cls.docker_directory_path):
       docker_tar = os.path.join('test_data', 'overlay2.v2.tgz')
-      tar = tarfile.open(docker_tar, 'r:gz')
-      tar.extractall('test_data')
-      tar.close()
+      with tarfile.open(docker_tar, 'r:gz') as tar:
+        tar.extractall('test_data')
```

Comment:
Does this require a tar.close()? I'm not familiar enough with the implementation.

Reply:
that's the whole point of Contex Manager, close() is called when __exit__() is called 
https://docs.python.org/3/reference/compound_stmts.html#with